### DISCUSSION TITLE
Opening projects/libraries just adds them to the list

### DISCUSSION THREAD

**Discussion by Kusselkopp** [flandreas/antares#342] [Ideas]
Is it correct that, currently, you cannot add existing projects and libraries to the list of projects/libraries in the explorer?
Every time I open a library I only see the library.

I think it would be better, if opening a library/project just adds this library/project to explorer list.

This would also allow moving a circuit from one project/library to another with a simple drag&drop.
E.g. if there is something that looks buggy but you don't want to export the whole project/library.

Btw. since several users might end up designing the same thing, would it be possible to implement a page/downloader for community libraries?
Like in KDE plasma where you can download different styles, e.g. for window or mouse, created and published by other users.

Btw. the Tanenbaum CPU download link is broken. :)

**Comment by flandreas** [comment]
Yes, in the current version, you can have 1 library and 0..1 project in the explorer. If you open the library, you can just work on the library. If you open a project (which is always based on a particular library), you can work on the project AND the library, also by moving circuit back and forth between the project and the library.

There is the function Library -> Edit Library which allows to you move circuits between two particular libraries.

But you are right, this is somehow too restrictive, and that's the reason for feature issue #212, which will allow you to have multiple open libraries, some of them you might have downloaded from a sharing site or so.

Thanks for pointing out the broken Tanenbaum example link. I will fix that. In the meantime, you can download it directly from https://github.com/flandreas/antares/releases?page=1. Expand the "Assets" tab in Release 0.17.0 and download the file directly.

**Comment by flandreas** [comment]
With release 1.0.0, the library structures have been reworked, and it is now possible to add multiple libraries to a single project.
